cfme/ansible_tower/explorer.py

```diff
@@ -121,16 +147,27 @@ class AnsibleTowerSystemsCollection(BaseCollection):
 
 
 @attr.s
-class AnsibleTowerJobTemplate(BaseEntity):
-    pass
+class AnsibleTowerJobTemplate(BaseEntity, Taggable):
+    name = attr.ib()
+    details_view = TowerExplorerJobTemplateDetailsView
 
 
 @attr.s
-class AnsibleTowerJobTemplatesCollection(BaseCollection):
+class AnsibleTowerJobTemplatesCollection(BaseCollection, TaggableCollection):
     ENTITY = AnsibleTowerJobTemplate
 
+    def all(self):
+        """Return entities for all items in Ansible Job templates collection"""
+        view = navigate_to(self, "All")
+        result = []
+        templates = view.entities.get_all()
+        for template in templates:
```

Comment:
optional:
as `name` is a single attribute and no other checks. I would return in a single line
```
return [self.instantiate(e) for e in view.entities.entity_names]
```
also, better to take care if multiple pages present.

Reply:
nice, I found `all_entity_names` that takes care of surfing all the pages